docker-compose.yml

```diff
@@ -2,10 +2,10 @@
 version: '3'
 services:
   app:
-    image: jasonwhite0/rudolfs:latest
-    #build:
-    #  context: .
-    #  dockerfile: Dockerfile
+    image: rudolfs:local
+    build:
+     context: .
+     dockerfile: Dockerfile
```

Comment:
Did you intend to commit these changes? Looks like this was just changed for local development.

Reply:
Oh, nope!

That is me being sloppy with version control.

I'll fix it.

Reply:
this should be sorted now